src/kontrol/kompile.py

```diff
@@ -75,6 +77,12 @@ def foundry_kompile(
         if regen or not req_path.exists():
             _LOGGER.info(f'Copying requires path: {req} -> {req_path}')
             shutil.copy(req, req_path)
+            # If the copied file is not writeable
+            if not os.access(req_path, os.W_OK):
+                # Fetch current permissions
+                current_permissions = os.stat(req_path).st_mode
+                # Grant write permissions
+                os.chmod(req_path, current_permissions | stat.S_IWUSR | stat.S_IWGRP | stat.S_IWOTH)
```

Comment:
Consider using `Path.chmod`.

Reply:
Done in https://github.com/runtimeverification/kontrol/pull/785/commits/597820377463129484ee18bd5a52df6855957d09, thanks!